Automattic/woocommerce-payments
# Register WC Pay nav items in WC navigation

Fixes https://github.com/woocommerce/woocommerce-admin/issues/5485

Note: this should probably not be merged in until https://github.com/woocommerce/woocommerce-admin/pull/5478 is approved and merged.

#### Changes proposed in this Pull Request

* Adds the navigation items for WooCommerce Payments in the new WooCommerce navigation


#### Screenshots

<img width="301" alt="Screen Shot 2020-10-28 at 1 21 44 PM" src="https://user-images.githubusercontent.com/10561050/97473472-64c24300-1921-11eb-9699-f94f4180d161.png">
<img width="300" alt="Screen Shot 2020-10-28 at 1 21 40 PM" src="https://user-images.githubusercontent.com/10561050/97473474-64c24300-1921-11eb-89c2-e534c5d8463c.png">
<img width="299" alt="Screen Shot 2020-10-28 at 1 20 57 PM" src="https://user-images.githubusercontent.com/10561050/97473475-655ad980-1921-11eb-9067-f662683a937e.png">
<img width="305" alt="Screen Shot 2020-10-28 at 1 14 55 PM" src="https://user-images.githubusercontent.com/10561050/97473476-655ad980-1921-11eb-800e-9282956527b5.png">


#### Testing instructions

1. Check that no regressions have occurred with WC Pay items.
1. Install WCA using the branch in https://github.com/woocommerce/woocommerce-admin/pull/5478
1. Enable the navigation under WooCommerce->Settings->Features.
1. Visit any WC Page.
1. Make sure the `Payments` category contains the "Connect" item if disconnected or all other items if stripe is connected.
1. Add another item to the `Payments` category.

```
			wc_admin_register_page(
				[
					'id'            => 'my-payment-item',
					'title'         => __( 'My payment item', 'text-domain' ),
					'parent'    => 'my-parent',
					'path'        => '/my-path',
					'category' => 'payments',
				]
			);
```
7. Make sure WC Pay items are put into a sub-category.
